docs/release.md

```diff
@@ -0,0 +1,108 @@
+# Quickstart New Image Playbook
+
+### 1. Build a new AMI using wardroom
+
+[Wardroom][wardroom] is a tool used to make Kubernetes-ready AMIs for use with
+the quickstart. The first step in a new quickstart image is building the image
+in Wardroom.
+
+* Checkout [`github.com/heptiolabs/wardroom`][wardroom].
+* You'll need Amazon creds in the environment. I recommend [aws-vault][vault].
+  If you use vault, preface the following commands with something like
+  `aws-vault exec <profile name> --`
+* Make you are on a clean tree, preferably the origin's `master`.
+  ```
+  $ git status
+  On branch master
+  Your branch is up to date with 'origin/master'.
+
+  nothing to commit, working tree clean
+  ```
+* Build the Packer image. [The repo has detailed instructions][packer], but the
+  basic command is something like `packer build -var-file us-east-1.json -var
+  kubernetes_version=1.9.2-00 -var kubernetes_cni_version=0.6.0-00 -var
+  build_version=$(git rev-parse HEAD) -only ami-ubuntu-16.04 packer.json`.
+* Boot an instance with the image you created, just to verify that everything
+  worked. If you have the [`aws` cli][aws], you can run something like: `aws
+  ec2 run-instances --image-id <your ami ID> --instance-type t1.micro --region
+  us-east-1 --key-name <aws keypair> --security-groups <security group>`. Make
+  sure `<security group>` allows SSH to your node. `aws ec2 describe-instances
+  --instance-ids i-02cedcd163ae3a8ba | grep -i public` will get you the IP/DNS
+  name. Then you can SSH in with `ssh -i <your pem file> ubuntu@<public ip or
+  DNS>`.
+* Make sure the instance has the version of kubernetes you expect, and any other
```

Comment:
What does "fix it" mean for a novice?

Reply:
I have no idea :( this is the point where a novice would have to dive deeper or ask one of us for help